src/index.ts

```diff
@@ -62,10 +42,36 @@ function getFileFromSuppliedFiles(filename: string, files: FetchHAROptions['file
   return false;
 }
 
-export default function fetchHAR(har: Har, opts: FetchHAROptions = {}) {
+export default async function fetchHAR(har: Har, opts: FetchHAROptions = {}): Promise<Response> {
```

Comment:
this function is now async since we're doing dynamic imports, but it has always returned a Promise<Response> so I **_think_** this should be a patch change?